lib/rails_admin_import/config/legacy_model.rb

```diff
@@ -0,0 +1,42 @@
+module RailsAdminImport
+  module Config
+    class LegacyModel
+      attr_reader :model_name
+      def initialize(model_name)
+        @model_name = model_name
+      end
+
+      def label(value)
```

Comment:
Unused method argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. You can also write as `label(*)` if you want the method to accept any arguments but don't care about them.